libs/gltfio/include/gltfio/FilamentInstance.h

```diff
@@ -49,6 +55,12 @@ class FilamentInstance {
     /** Gets the transform root for the instance, which has no matching glTF node. */
     utils::Entity getRoot() const noexcept;
 
+    /** Gets the NameComponentManager label for the given entity, if it exists. */
+    const char* getName(utils::Entity) const noexcept;
+
+    /** Gets the application-specific extras loaded from asset for specific entity, if such exist. */
+    std::string getExtras(utils::Entity entity) const noexcept;
```

Comment:
Please don't expose STL types. Return const char* instead, or our own CString (although I don't know if it's public).

Reply:
Removed